src/content/blog/astro-350.mdx

```diff
@@ -0,0 +1,264 @@
+
+Enable the experimental routing option by adding an `i18n` object to your Astro configuration with a default location and a list of all languages to support:
+
+```js
+// astro.config.mjs
+import {defineConfig} from "astro/config";
+
+export default defineConfig({
+  experimental: {
+    i18n: {
+      defaultLocale: "en",
+      locales: ["en", "es", "pt-br"]
+    }
+  }
+})
+```
+
+Organize your content folders by locale depending on your `i18n.routingStrategy`, and Astro will handle generating your routes and showing your preferred URLs to your visitors.
+```
+├── src
+│   ├── pages
+│   │   ├── about.astro
+│   │   ├── index.astro
+│   │   ├── es
+│   │   │   ├── about.astro
+│   │   │   ├── index.astro
+│   │   ├── pt-br
+│   │   │   ├── about.astro
+│   │   │   ├── index.astro
+```
+
+Compute relative URLs for your links with `getLocaleRelativeURL` from the new `astro:i18n` module:
+
+```astro
+---
+import {getLocaleRelativeUrl} from "astro:i18n";
+const aboutUrl = getLocaleRelativeUrl("pt-br", "about");
+---
+<p>Learn more <a href={aboutURL}>About</a> this site!</p>
+```
+
+Enabling i18n routing also provides two new properties for browser language detection: `Astro.preferredLocale` and `Astro.preferredLocaleList`. These combine the browser's `Accept-Language` header, and your site's list of supported languages and can be used to automatically respect your visitor's preferred languages.
+
+Read more about Astro's [experimental i18n routing](https://docs.astro.build/en/guides/internationalization/) in our documentation.
+
+## Prefetch
+
+[Prefetch](https://developer.mozilla.org/en-US/docs/Glossary/Prefetch) is a browser feature where you can fetch a page that you think your user is likely to visit, and when they go to navigate to that page it will be (partially loaded). For multi-page apps, prefetch is an important part of keeping your site speedy.
+
+In Astro, in the past prefetch has been available via an official integration (`@astrojs/prefetch`). Today we are bringing pretech into core and expanding what it can do.
+
+You can enable prefetching for your site with the `prefetch: true` config. It is enabled by default when using [View Transitions](https://docs.astro.build/en/guides/view-transitions/) and can also be used to configure the `prefetch` behaviour used by View Transitions.
+
+You can enable prefetching by setting `prefetch:true` in your Astro config:
+
+```js 
+// astro.config.js
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  prefetch: true
+})
+```
+
+In addition to being a core feature, the new prefetch also comes with more flexibility:
+
+- You can define if prefetch occurs on `tap` (a touch that's not a full click), `hover` and `viewport` (when the element is in view).
```

Comment:
I guess since we're listing the options, it should rather be "or", and also comma nit.
```suggestion
- You can define if prefetch occurs on `tap` (a touch that's not a full click), `hover`, or `viewport` (when the element is in view).
```